docs/API_SPECS_SIMULATIONS.md

```diff
@@ -472,62 +512,75 @@ Content-Type: application/json; charset=utf-8
 ```
 ```json
 {
-  "ETag": "969ee1fb277640",
-  "Id": "1",
-  "Name": "Sample Simulation",
-  "Description": "This is a sample simulation",
-  "Enabled": false,
-  "Running": false,
-  "IoTHubs": [
+   "ETag":"969ee1fb277640",
```

Comment:
the actual ETag has quotes, which could be confusing, so either we leave it out or provide a real example - your call